configs/platforms/solaris-11-i386.rb

```diff
@@ -40,7 +41,11 @@
 basedir=default" > /var/tmp/vanagon-noask;
   echo "mirror=https://artifactory.delivery.puppetlabs.net/artifactory/generic__remote_opencsw_mirror/testing" > /var/tmp/vanagon-pkgutil.conf;
   pkgadd -n -a /var/tmp/vanagon-noask -d http://get.opencsw.org/now all
+<<<<<<< HEAD
   /opt/csw/bin/pkgutil --config=/var/tmp/vanagon-pkgutil.conf -y -i libffi_dev autoconf gcc4core || exit 1;
+=======
+  /opt/csw/bin/pkgutil --config=/var/tmp/vanagon-pkgutil.conf -y -i libffi_dev autoconf gcc4core|| exit 1;
+>>>>>>> (PA-4872) OpenSSL 3 work for Solaris-11-x86-64
```

Comment:
ah looks like this conflict still needs resolving, note the `gcc4core ||` vs `gcc4core||` difference